car_control.py

```diff
@@ -171,31 +162,89 @@ def update(self, line):
             self.throttle_old_result = throttle_new_result
     
             throttle_p_output = throttle_new_result # Standard PID Stuff here... nothing particularly interesting :)
-            self.throttle_i_output = max(min(self.throttle_i_output + throttle_new_result, THROTTLE_I_MAX), THROTTLE_I_MIN)
+            self.throttle_i_output = max(min(self.throttle_i_output + throttle_new_result, self.THROTTLE_I_MAX), self.THROTTLE_I_MIN)
             throttle_d_output = ((throttle_delta_result * 1000) / self.fps.delta()) if self.fps.delta() else 0
-            throttle_pid_output = (THROTTLE_P_GAIN * throttle_p_output) + \
-                                  (THROTTLE_I_GAIN * self.throttle_i_output) + \
-                                  (THROTTLE_D_GAIN * throttle_d_output)
+            throttle_pid_output = (self.THROTTLE_P_GAIN * throttle_p_output) + \
+                                  (self.THROTTLE_I_GAIN * self.throttle_i_output) + \
+                                  (self.THROTTLE_D_GAIN * throttle_d_output)
     
             # Throttle goes from 0% to 100%.
             self.throttle_output = max(min(throttle_pid_output, 100), 0)
         else:
             self.throttle_output = self.throttle_output * 0.99
-    
+            self.steering_output = self.STEERING_OFFSET
+            
+        self.jsonreadout()
```

Comment:
Can you add a quick comment here saying what this is meant to do and why it is here?

Reply:
yes, we are working on adding better commenting to our additions to the code.